pkgs/development/python-modules/drawille/default.nix

```diff
@@ -0,0 +1,19 @@
+{lib, buildPythonPackage, fetchPypi}:
+
+buildPythonPackage rec {
+  pname = "drawille";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-t4nS8TWbEGKHibIbLfZZycPQxTiEzuJ7DYsa6Twi+8s=";
+  };
```

Comment:
We really want to run tests for python packages and if the pypi packages does not include the `tests.py` file we should fetch from GitHub instead.

Then use `unittestCheckHook` in `nativeCheckInputs`.

Please look at enabling tests for all python packages, should they provide any. If they don't please set up `pythonImportsCheck` instead.

Reply:
The problem here:

Neither `drawille` nor `drawilleplot` have tags on the GH repos, therefore I can't know which commit is the actual release.

`drawille` seem to have some tests in the repo, but not in the pypi release.
`drawilleplot` doesn't have tests at all.

I will therefore leave both using `fetchPypi` and then set up the import check.

Reply:
Create an issue to notify upstream about this pickle.

Reply:
You'll also want to disable the check phase by setting `doCheck = false;`.

Reply:
Last commit has been years ago on both repos, seem to be abondoned…